corehq/apps/sms/handlers/keyword.py

```diff
@@ -65,6 +68,48 @@ def handle_global_keywords(v, text, msg, text_words, open_sessions):
     return fcn(v, text, msg, text_words, open_sessions)
 
 
+def global_keyword_update(v, text, msg, text_words, open_sessions):
+
```

Comment:
We should check to make sure that `v.owner_doc_type` is CommCareUser, and if not, we should reply with the unrecognized update action message